tools/qtc_templates/kuesa/project/empty_quick/app.pro

```diff
@@ -0,0 +1,19 @@
+QT += quick
+CONFIG += resources_big
+
+DEFINES += QT_DEPRECATED_WARNINGS
+
+SOURCES += \\
```

Comment:
Shouldn't that be just one \ ?

In fact, I'm not quite sure how qmake would deal with a double one here. :-)

Reply:
done